packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -234,6 +234,21 @@ export class MLSService extends TypedEventEmitter<Events> {
     return this.apiClient.api.conversation.deleteSubconversation(conversationId, SUBCONVERSATION_ID.CONFERENCE, data);
   }
 
+  public async leaveConferenceSubconversation(conversationId: QualifiedId): Promise<void> {
+    const subconversationGroupId = await this.getGroupIdFromConversationId(
+      conversationId,
+      SUBCONVERSATION_ID.CONFERENCE,
+    );
+
+    const isSubconversationEstablished = await this.conversationExists(subconversationGroupId);
```

Comment:
don't we want to capitalize `Conversation` in `isSubConversationEstablished`?

Reply:
I think we treat `subconversation` as a single word 😅

Reply:
I looked at the rest of the file, yep, it's well established by now